src/components/tabs/js/tabsController.js

```diff
@@ -163,6 +163,9 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
 
   function handleMaxTabWidth (newWidth, oldWidth) {
     if (newWidth !== oldWidth) {
+      angular.forEach(elements.tabs, function(tab) {
+        tab.style.maxWidth = newWidth + 'px';
```

Comment:
Does this update if the max-width changes? 

(I'm not actually sure if it needs to. cc @robertmesserle)